troposphere/static/js/components/settings/AdvancedSettingsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+
 import SSHConfiguration from "components/settings/advanced/SSHConfiguration";
+import ClientCredentials from "components/settings/advanced/ClientCredentials";
```

Comment:
Both [ExportCredential.jsx](https://github.com/cyverse/troposphere/blob/undulating-umbrellabird/troposphere/static/js/components/modals/ExportCredential.jsx) and [ClientCredentials.jsx](https://github.com/cyverse/troposphere/blob/undulating-umbrellabird/troposphere/static/js/components/settings/advanced/ClientCredentials.jsx) were introduced in the pull request #508 (and tweaked in #509 & #513).